kcidev/libs/common.py

```diff
@@ -129,8 +129,8 @@ def kci_msg_nonl(content):
     click.echo(content, nl=False)
 
 
-def kci_msg_bold_nonl(content):
```

Comment:
This change is part of the common library common print system, and not directly related with maestro validate builds. Would be better to be in a different commit for improve the commit readability.

Reply:
I need to include this in the same commit otherwise it would break `print_simple_list` function from `validate/helper.py` function which used the `kci_msg_bold_nonl`.

Reply:
right, you have to reverse the operation for changing it before this commit.

Reply:
even if changed before the current commit?